### PR TITLE
[#143982623] Customer.io update customer worker

### PR DESCRIPTION
**Pull request by sergii-tkachenko**
#### What's this PR do?
- Creates `customer-io-update-customer` worker
- Creates `CustomerIoIdentifyMessage`
- Transforms `UserMessage` to `CustomerIoIdentifyMessage`

#### How should this be manually tested?
- `yarn install`
- `docker-compose up`
- `yarn all-tests`
- `yarn web`
- `yarn worker customer-io-update-customer`

#### Next Up
- Test coverate

#### What are the relevant tickets?
Pivotal [143982623](https://www.pivotaltracker.com/story/show/143982623)
Gateway-PHP integration: https://github.com/DoSomething/gateway/pull/74

### CODE REVIEW

File: src/messages/CustomerIoIdentifyMessage.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+        // When interests not present, make them an empty array.
+        interests: Joi.array().items(Joi.string()).empty(null).default(undefined),
+
+        // TODO: add cio specific fields, like unsubscribe
+      }),
+      // TODO: Bring back when phone is formatted in Northstar
+      // .or('email', 'phone'),
+    });
+  // Require presence at least one of: keyword, args, mms_image_url.
```
Comment (rapala61) [inline src/messages/CustomerIoIdentifyMessage.js]: should be removed?

File: src/messages/CustomerIoIdentifyMessage.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  static fromUser(userMessage) {
+    const user = userMessage.payload.data;
+    // Copy user fields.
+    const customerData = Object.assign({}, user);
+    // Remove id from data, as it's available on top level.
+    delete customerData.id;
+
+    // Rename mobilecommon_status to mobile_status
+    if (customerData.mobilecommon_status) {
+      const mobileStatus = customerData.mobilecommon_status;
```
Comment (rapala61) [inline src/messages/CustomerIoIdentifyMessage.js]: Making a `renaming` function could makes sense here.

File: src/messages/CustomerIoIdentifyMessage.js (hunk 1 above)
Comment (sergii-tkachenko) [inline src/messages/CustomerIoIdentifyMessage.js]: Yes.

File: src/messages/CustomerIoIdentifyMessage.js (hunk 2 above)
Comment (sergii-tkachenko) [inline src/messages/CustomerIoIdentifyMessage.js]: Yes, but I'll leave it to the next iteration.

**Pull request merged**